dstep/translator/MacroDefinition.d

```diff
@@ -2053,6 +2053,10 @@ bool translateFunctAlias(
     {
         Identifier ident = cast(Identifier) expr.expr;
 
+        import dstep.core.Traits : isKeyword;
+        if (ident.spelling.isKeyword())
+            return false;
```

Comment:
In other places in the code, when there's a symbol with is a D keyword that is translate, an underscore is added as a suffix. Which there is a separate function for [1].

[1] https://github.com/jacob-carlborg/dstep/blob/master/dstep/translator/Translator.d#L469